thirdeye/thirdeye-frontend/app/styles/components/te-anomaly-table.scss

```diff
@@ -145,4 +145,15 @@
   &--performance {
     margin-top: 30px;
   }
+
+  &__redirect-link {
+    display: flex;
+    justify-content: center;
+    border: solid 1px $te-light-grey;
+    margin-top: -21px;
```

Comment:
I'd rather not use magic numbers with margin/padding for positioning. 
What is the intention here? Let's discuss this on Monday

Reply:
The intention is for the `See More` row to be adjacent to the bottom of the table, as opposed to having a gap between the two. I tried setting the table's margin and padding and border bottom to be `0px`, but there was still some gap that I'm not sure what is causing it. We can talk about this more in person.